shepherd_bms/src/stateMachine.cpp

```diff
@@ -11,7 +11,15 @@ StateMachine::~StateMachine()
 
 void StateMachine::initBoot()
 {
-    return;
+    AccumulatorData_t *prevAccData = nullptr;
+    uint32_t bmsFault = FAULTS_CLEAR;
+    uint16_t overVoltCount = 0;
+    uint16_t underVoltCount = 0;
+    uint16_t overCurrCount = 0;
+    uint16_t chargeOverVolt = 0;
+    uint16_t overChgCurrCount = 0;
+    uint16_t lowCellCount = 0;
+    analyzer.push(prevAccData);
```

Comment:
i dont think we want to define these within this function, probably just set them and declare them in statmachine.h

Reply:
These don't look like they are defined in the function @dyldonahue

Reply:
commit was made after that comment i believe